### PR TITLE
Include asciimath filter

### PR DESCRIPTION
**Pull request by Kerl13**

### CODE REVIEW

**Comment by Kerl13** [comment]
The last commits aim at : 
- Use the last - and the cleanest - version of AsciiMath
- Update the makefile
  - to work with AsciiMath
  - to have a better behaviour when you edit a `.md` source file

So, there is no major change excepted the integration of AsciiMath

File: Makefile [hunk 1]
```diff
@@ -1,29 +1,34 @@
 SOURCES = $(wildcard *.md)
 OUT = $(SOURCES:.md=.pdf)
+ASCIIMATH=filters/asciimath/pandoc-asciimath
 
-HS_FILTERS_NAMES = nice-frac usual-fun
```
Comment (Phyks) [inline Makefile:4]: TODO: Remove associated filters as well.

File: Makefile [hunk 2]
```diff
@@ -1,29 +1,34 @@
...
+HS_FILTERS_NAMES =
 PY_FILTERS_NAMES = pandoc-svg.py
-EXT_FILTERS = pandoc-crossref
```
Comment (Phyks) [inline Makefile:6]: Why deleted?

File: notes.md [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
 cref: True
 chapters: True
 ---
 
 \pagebreak
 
+# AsciiMath example
+
+Soit $n in NN$, alors on a
+$$ sum_(k=1)^n k^3 = ((n(n+1))/2)^2 $$
```
Comment (Phyks) [inline notes.md:16]: English text?

**Pull request merged**